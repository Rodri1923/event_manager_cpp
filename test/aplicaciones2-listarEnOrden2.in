# se asume un ID máximo de persona de 10

crearTTablaJugadores 20

# -- jugador1, id 1, 10 años --
crearJugada
1 1 2 3 4
agregarEnPartida
crearJugador 1 10 Uruguay jugador1
insertarJugadorEnTabla

# -- jugador4, id 4, 40 años --
crearJugada
1 1 1 1 1
agregarEnPartida
crearJugador 4 40 Uruguay jugador4
insertarJugadorEnTabla

# -- jugador5, id 5, 50 años
crearJugada
1 4 3 2 1
agregarEnPartida
crearJugador 5 50 Brasil jugador5
insertarJugadorEnTabla

# -- jugador7, id 7, 70 años
crearJugada
1 1 1 1 1
agregarEnPartida
crearJugador 7 70 Uruguay jugador7
insertarJugadorEnTabla


# -- jugador8, id 8, 80 años
crearJugada
1 3 3 3 3
agregarEnPartida
crearJugador 8 80 Uruguay jugador8
insertarJugadorEnTabla

# -- jugador2, id 2, 20 años
crearJugada
1 1 1 1 1
agregarEnPartida
crearJugador 2 20 Uruguay jugador2
insertarJugadorEnTabla

# -- jugador10, id 10, 100 años
crearJugada
2 2 2 2 2
agregarEnPartida
crearJugador 10 100 Uruguay jugador10
insertarJugadorEnTabla

# -- jugador3, id 3, 30 años
crearJugada
1 1 1 1 1
agregarEnPartida
crearJugador 3 30 Uruguay jugador3
insertarJugadorEnTabla


# el orden debe ser 1, 3, 4, 5, 7, 10
listarEnOrden 6 jugador5 jugador7 jugador1 jugador3 jugador10 jugador4

liberarTTablaJugadores

Fin
