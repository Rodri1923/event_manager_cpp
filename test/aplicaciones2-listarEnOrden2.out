1>#  se asume un ID máximo de persona de 10.
2>3>#  -- jugador1, id 1, 10 años --.
4>5>6>7>8>#  -- jugador4, id 4, 40 años --.
9>10>11>12>13>#  -- jugador5, id 5, 50 años.
14>15>16>17>18>#  -- jugador7, id 7, 70 años.
19>20>21>22>23>#  -- jugador8, id 8, 80 años.
24>25>26>27>28>#  -- jugador2, id 2, 20 años.
29>30>31>32>33>#  -- jugador10, id 10, 100 años.
34>35>36>37>38>#  -- jugador3, id 3, 30 años.
39>40>41>42>43>#  el orden debe ser 1, 3, 4, 5, 7, 10.
44>Jugador 1: jugador1, Uruguay, 10 años
Jugada: 1 
Movimientos: Derecha Izquierda Arriba Abajo 
Jugador 3: jugador3, Uruguay, 30 años
Jugada: 1 
Movimientos: Derecha Derecha Derecha Derecha 
Jugador 4: jugador4, Uruguay, 40 años
Jugada: 1 
Movimientos: Derecha Derecha Derecha Derecha 
Jugador 5: jugador5, Brasil, 50 años
Jugada: 1 
Movimientos: Abajo Arriba Izquierda Derecha 
Jugador 7: jugador7, Uruguay, 70 años
Jugada: 1 
Movimientos: Derecha Derecha Derecha Derecha 
Jugador 10: jugador10, Uruguay, 100 años
Jugada: 2 
Movimientos: Izquierda Izquierda Izquierda Izquierda 
45>46>Fin.
